hydroflow_plus/src/cycle.rs

```diff
@@ -6,15 +6,15 @@ use crate::builder::Builders;
 use crate::node::HfNode;
 use crate::HfStream;
 
-pub struct HfCycle<'a, T, N: HfNode<'a>> {
+pub struct HfCycle<'a, T, W, N: HfNode<'a>> {
```

Comment:
Would be good to either give these type parameters longer self-explanatory names, or include a doc comment saying what they mean